### PR TITLE
James 1818 Remove store imports in test classes

### PR DESCRIPTION
**Pull request by rouazana**
Don't worry, only the last commit is to review in this PR

### CODE REVIEW

**Comment by chibenwa** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

**Comment by rouazana** [comment]
Sorry, actually 2 commits.

File: mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
 import org.apache.james.mailbox.inmemory.InMemoryMailboxSessionMapperFactory;
```
Comment (aduprat) [inline mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java]: The title of this commit seems to be not appropriated.

Comment (rouazana) [inline mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java]: Why? Oh I see... I changed the name in the same time, but as you can see below (its not very clear in github) the Authenticator has been moved. That's the essential point of this commit.

**Comment by mbaechler** [comment]
👍

**Comment by aduprat** [comment]
:+1:

**Pull request merged**